docs/reference/api/security/TLS.md

```diff
@@ -41,6 +41,26 @@ Then create a file named `mbed_app.json` at the root of your application with th
 }
 ```
 
+### Mbed TLS platform context
+
+Some hardware accelerators require initialization, regardless of the specific cryptography engine. For this, we introduced `mbedtls_platform_setup()` and `mbedtls_platform_terminate()`.
+
+As the [examples](#mbed-tls-examples) show, you *must* call the `mbedtls_platform_setup()` function before you call any Mbed TLS API. After using the Mbed TLS API, you *must* call `mbedtls_platform_teardown()` to terminate the platform hardware driver. We suggest you set the platform context parameter for these functions as `NULL`.
+
+```
+    int exit_code = MBEDTLS_EXIT_FAILURE;
```

Comment:
The exit code variable has a mixed meaning in the example: the return code for the program and for the Mbed TLS API. It has been demonstrated that Mbed TLS return codes exceed allowed exit codes for some platforms (e.g. POSIX), and therefore it may be advisable not to return them from Mbed TLS programs. I think, that this example can be expressed without the `exit_code` variable. Instead you could only declare `ret` to capture the Mbed TLS error code.

Reply:
ok

Reply:
Note that this was taken from our examples, do you think we should change them?

Reply:
I don't think the examples need to be changed regarding this. The `exit_code` is set correctly there, but it is done in places that you had to cut away for this example.